### DISCUSSION TITLE
Frequency in .json?

### DISCUSSION THREAD

**Discussion by Crsarmv7l** [merbanan/rtl_433#2511] [Q&A] [answered]
Is there a way to have the frequency output in .json? I am looking to have rtl_433 hop and output to a json and it would be helpful to know what freqs received signals were seen on.

**Comment by zuckschwerdt** [comment, marked as answer]
Frequency is in the meta-data for `-M level`
E.g. `rtl_433 -M level -M time:usec -F json`
```
{"time" : "2023-05-26 23:21:44.542511", "model" : "LaCrosse-TX141THBv2",
 "id" : 126, "channel" : 1, "battery_ok" : 1, "temperature_C" : 19.100,
 "humidity" : 55, "test" : "No", "mic" : "CRC", "mod" : "ASK",
 "freq" : 433.968, "rssi" : -0.207, "snr" : 19.875, "noise" : -20.082}
```

**Reply by Crsarmv7l** [reply]
Thank you so much yet again for your awesome help!

**Comment by Crsarmv7l** [comment]
Is there a way to omit the hops from being in the json? Seem like there is a json entry for every hop.
